server/index.js

```diff
@@ -108,6 +108,7 @@ app.get('/qa/questions', (req, res) => {
     }
   })
     .then(data => {
+      // console.log('testResponse', data.data.results[0].answers)
```

Comment:
Consider just getting rid of the commented out console.logs eventually, after you've finished using them for testing purposes.